homeassistant/components/progettihwsw/__init__.py

```diff
@@ -0,0 +1,62 @@
+"""Automation manager for boards manufactured by ProgettiHWSW Italy."""
+import asyncio
+
+from ProgettiHWSW.ProgettiHWSWAPI import ProgettiHWSWAPI
```

Comment:
@ArdaSeremet do you have this package available on github?

Reply:
I published it to GitHub.